cogs/mappinghelp.py

```diff
@@ -0,0 +1,83 @@
+# Std Lib Imports
```

Comment:
what does this cog do?

Reply:
this is the start to commands used in the mapping help forums. people have been asking for a /close command to auto mark as solved and close it